CONTRIBUTING.md

```diff
@@ -1,7 +1,51 @@
 # Summary
+
 Contributing to the Mongodb Controllers for Kubernetes (MCK) project
-Pull requests are always welcome, and the MCK dev team appreciates any help the community can give to help make MongoDB better.
+Pull requests are always welcome, and the MCK dev team appreciates any help the community can give to help make MongoDB
+better.
+
+## PR Prerequisites
 
-# PR Prerequisites
-* Please ensure you have signed our Contributor Agreement. You can find it [here](https://www.mongodb.com/legal/contributor-agreement).
+* Please ensure you have signed our Contributor Agreement. You can find
+  it [here](https://www.mongodb.com/legal/contributor-agreement).
 * Please ensure that all commits are signed.
+* Create a changelog file that will describe the changes you made. Use the `skip-changelog` label if your changes do not
+  require a changelog entry.
+
+## Changelog files and Release Notes
+
+Each Pull Request usually has a changelog file that describes the changes made in the PR using Markdown syntax.
+Changelog files are placed in the `changelog/` directory and used to generate the Release Notes for the
+upcoming release. Preview of the Release Notes is automatically added as comment to each Pull Request.
```

Comment:
How do we generate release notes after looking at the change log? I am sorry if it's already documented somewhere and I don't remember. 
Or in other words, do we polish/rephrase the changelog to make it release note or just use the change log as release note?